src/core/ext/filters/client_channel/lb_policy/xds/xds.cc

```diff
@@ -1823,34 +1540,45 @@ class XdsFactory : public LoadBalancingPolicyFactory {
     }
     std::vector<grpc_error*> error_list;
     // Child policy.
-    RefCountedPtr<LoadBalancingPolicy::Config> child_policy;
+    Json child_policy_json;
     auto it = json.object_value().find("childPolicy");
-    if (it != json.object_value().end()) {
-      grpc_error* parse_error = GRPC_ERROR_NONE;
-      child_policy = LoadBalancingPolicyRegistry::ParseLoadBalancingConfig(
-          it->second, &parse_error);
-      if (child_policy == nullptr) {
-        GPR_DEBUG_ASSERT(parse_error != GRPC_ERROR_NONE);
-        std::vector<grpc_error*> child_errors;
-        child_errors.push_back(parse_error);
-        error_list.push_back(
-            GRPC_ERROR_CREATE_FROM_VECTOR("field:childPolicy", &child_errors));
-      }
+    if (it == json.object_value().end()) {
+      child_policy_json = Json::Array{Json::Object{
+          {"round_robin", Json::Object()},
+      }};
+    } else {
+      child_policy_json = it->second;
```

Comment:
Ditto. Can we use pointer instead to avoid possible copy?

Reply:
Done.